docs/concepts/true-hybrid-search.mdx

```diff
@@ -0,0 +1,266 @@
+---
+title: "True Hybrid search"
+---
+
+True Hybrid Search allows you to combine multiple retrieval techniques — such as multi-vector search, hybrid (vector \+ text) search
+sort the results using a custom scoring function **within a single query**.
+
+## How TopK differs from other "hybrid" search systems
+
+Most databases that offer "hybrid" search maintain separate vector and keyword indexes. When a query is executed:
+
+1. They run the query against both indexes separately
+2. After, each index returns its top results (e.g. first 100 candidates)
+3. Lastly, use techniques like Reciprocal Rank Fusion (RRF) to merge and rerank these two sets of results
+
+<br />
+
+```mermaid
```

Comment:
question: can we "center" these? looks off when they are on the left

Reply:
yes, need to use custom css